pkg/apiserver/statement/service.go

```diff
@@ -29,6 +29,11 @@ import (
 	"github.com/pingcap-incubator/tidb-dashboard/pkg/tidb"
 )
 
+var (
+	ErrNS     = errorx.NewNamespace("error.api.export")
```

Comment:
Better to call it ErrNS (with code == `error.api.statement`) and ErrExportNoData (with type `export_no_data`) since it is inside the Statement API and export feature is only a small part of it. Later when other people want to introduce a new error, the current error NS code will be misleading.

Reply:
Originally I thought of the export feature as a module under the root level so use `error.api.export` namespace. But considering the confusion it may cause, I will change it.